website/tutorial/managing-your-application-resources.md

```diff
@@ -0,0 +1,198 @@
+3. Use the **Stop** icon to stop a container or pod.
+4. Use the **Delete** icon to delete a container or pod.
+
+   :::note
+
+   - You cannot delete a container if it is being used in any pods. In such cases, you must delete the pod and then delete the container.
+
+   :::
+
+5. Bulk deletion: Select containers or pods and delete them by using the **Delete** button.
+6. Click the name of the container or pod to:
+   - View its summary and logs.
+   - View its kube configuration if the container or pod is a Kubernetes resource.
+   - Interact with the container using a terminal
+7. View containers or pods based on their status:
+
+   - Select the **Running** tab to view all running containers or pods
+   - Select the **Stopped** tab to view all stopped containers or pods
+
+8. Use the **Prune** button to remove all unused containers or pods from the Podman engine.
+
+**_Actions specific to containers_**
+
+![container actions](img/container-actions.png)
+
+1. Use the **Create** button to create a container using a container or docker file or from an existing image.
+2. Use the **Create Pod** button after selecting containers to create a pod.
+3. Use the overflow menu to:
+
+   - Open and view logs.
+   - Kubernetes action: Generate and view configuration for a Kubernetes container.
+   - Kubernetes action: Deploy a container to a Kubernetes cluster.
+   - Kubernetes action: Open a browser to check the cluster status.
+   - Open a terminal to interact with the container.
+   - Restart your container.
+   - Export a container to your local machine.
+
+   :::note
+
+   You might see varying options in the overflow menu based on status and configuration of the container.
+
+   :::
+
+**_Actions specific to pods_**
+
+![pod actions](img/pods-actions.png)
+
+1. View the status of each container by hovering over it in the _CONTAINERS_ column. The status can be `Running`, `Created`, `Exited`, or `Waiting`.
+2. Use **Play Kubernetes YAML** to create a pod from a Kubernetes YAML file.
+3. Use the overflow menu to:
+   - Kubernetes action: Generate and view configuration for a Kubernetes pod.
+   - Kubernetes action: Deploy a pod to a Kubernetes cluster.
+   - Restart your pod.
+
+## Managing images
+
+Perform any of the following actions after navigating to the component page.
+
+![image actions](img/image-actions.png)
+
+1. Search an image using the search box.
+2. Use the **Run** icon to create a container from the image.
+3. Use the **Delete** icon to delete an image.
+
+   :::note
+
+   - You cannot delete an image if it is being used in any containers. In such cases, you must delete the container and then delete the image.
+   - By selecting multiple images, you can delete or save them in a single step.
+     :::
+
+4. Use the overflow menu to:
+
+   - Push an image to a configured registry.
+   - Edit the image name and tag.
+   - View history of the image to view the commands that were used to create each layer within an image.
```

Comment:
This sentence is very confusing to read to me. Maybe we can rephrase it. 

Suggestion take it or leave it: "View history of the image and the commands that created each layer of the image"

Reply:
Changed it to: View history to check the commands that created each layer of the image. This makes the sentence concise and clear.